CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Some guidelines in reading this document:
 * Being that these are the early days of the repository, we have some code changes that were added directly and without much detail, for these we have a link to the commit instead of the PR.
 * Annotations starting with **[BC]** indicates breaking change.
 
+## [new release]
+
+* **[BC]** `selectQuery` renamed to `selectRequest` and now also accepts `{ cacheID, page }` for selecting Requests.
+* Fix the Promise return from middleware dispatch ([#12](https://github.com/log-oscon/redux-wpapi/issues/12))
```

Comment:
When resolving issues we should probably have a structure more like:
```
* Fixes #[number] – some other text after with extra information... (#[number_of_PR])
```

This way the link to the issue comes first always, while at the end maintaining the PR link.

Furthermore, for a PR with several "bullets" we should do something like this:

```markdown
* [generic option description/title] (#[PR_link])
    * Individual noteworthy change
    * **BC** Individual noteworthy breaking change
    * [...]
```